docs/content/tutorials/composite-recipe/index.md

```diff
@@ -0,0 +1,180 @@
+
+Composite Recipes are only written in Bicep. If you used Terraform in the previous tutorial for your Recipes, you will need an OCI registry to store your Recipe. While Terraform-based Recipes are stored in Git, Bicep-based Recipes can only be published to OCI registry.
+
+## Step 1: Create the web service resource type
+
+Create or modify the `types.yaml` file so that it has both the `postgreSQL` type from the previous tutorial and the new `webServices` type. The schema for the new web services resource type is lengthy, so it is best to download the fully prepared file with both resource types.
+
+{{< button text="Download types.yaml" link="snippets/types.yaml" newtab="true" >}}
+
+This web services schema is simpler than it appears:
+
+* `environment` and `application` have the same purpose as in the previous PostgreSQL example. These properties are on all resource types.
+* `connections` is used to create dependencies, see [How-To: Connect to dependencies]({{< ref "guides/author-apps/containers/howto-connect-dependencies" >}}) for more details
+* `ingress` is the first property the developer can set. The purpose of this property is to expose an option for developers to specify if the web service is accessible to connections from outside the cluster. This functionality will be implemented in the Recipe in step 3. 
+* `container` is a duplication of the Containers schema. Since the web service resource type extended Containers, the original schema must be included here. As you will see when implementing the Recipe, duplicating the Containers schema exactly makes it simple to pass multiple properties in only a few lines of code.
+
+Create the resource type using the [rad resource-type create]({{< ref rad_resource-type_create >}}) command.
+
+```bash
+rad resource-type create webServices -f types.yaml
+```
+You can confirm the resource type was created using `rad resource-type list` command.
+
+## Step 2: Update the Bicep extension
+
+Generate the Bicep extension using the [rad bicep publish-extension]({{< ref rad_bicep_publish-extension >}}) command.
+
+```bash
+rad bicep publish-extension -f types.yaml --target radiusResources.tgz
+```
+
+Since the `bicepconfig.json` file was modified in the previous tutorial, it does not need to be further modified.
+
+## Step 3: Create, publish, and register the composite Recipe
+
+The previous tutorial demonstrated deploying a resource using Terraform or Bicep resource providers such as the Kubernetes provider. This tutorial creates a composite Recipes which uses only built-in Radius types. These composite Recipes are only possible using Bicep.
+
+1. Create a new file called `webservices.bicep` and add the following:
+
+   {{% rad file="snippets/webservices.bicep" lang="bicep" embed=true %}}
```

Comment:
this is not rendering on the page: 

<img width="852" alt="image" src="https://github.com/user-attachments/assets/761d1dbd-d79f-445e-a2a0-8834efa51137" />